utils/xmtpRN/conversations.ts

```diff
@@ -312,6 +312,13 @@ const listGroups = async (client: ConverseXmtpClientType) => {
 
 export const importedTopicsDataForAccount: { [account: string]: boolean } = {};
 
+// Helper function to compare two sets
+function areSetsEqual(setA: Set<any>, setB: Set<any>): boolean {
```

Comment:
Best to put this in a separate file

Reply:
Also normally better practice to use Set<unknown> rather than any it will force better checks